src/main/java/org/openlmis/buq/domain/buq/BottomUpQuantificationStatusChange.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org.
+ */
+
+package org.openlmis.buq.domain.buq;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.PrePersist;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.openlmis.buq.domain.BaseEntity;
+
+@Entity
+@Table(name = "bottom_up_quantification_status_changes")
+@NoArgsConstructor
+@AllArgsConstructor
+public class BottomUpQuantificationStatusChange extends BaseEntity {
+
+  @ManyToOne(cascade = {CascadeType.REFRESH})
+  @JoinColumn(name = "bottomupquantificationId", nullable = false)
+  @Getter
+  @Setter
+  private BottomUpQuantification bottomUpQuantification;
+
+  @NotNull
+  @Getter
+  @Setter
+  private UUID authorId;
+
+  @Column(columnDefinition = "timestamp with time zone")
+  @Getter
+  @Setter
+  private ZonedDateTime occurredDate;
```

Comment:
I  wonder if this is  the same use case as `BaseTimestampedEntity` covers.

Reply:
I decided to make a separate field instead of using BaseTimestampedEntity because BUQ Status Change stores information about a certain event in time, the modification of which does not make sense and should not be possible.